src/cls/_ZPM/Storage/ModuleTemplate.cls

```diff
@@ -142,10 +142,10 @@ ClassMethod Expfilename(code, wd)
 }
 
 /// returns list of non-system csp applications in the current namespace
-ClassMethod GetCSPApplications(Output pApplications As %List, currentNamespace = {$Namespace}) As %Status
+ClassMethod GetCSPApplications(Output pApplications As %List) As %Status
 {
     Set pApplications = ""
-    
+    Set currentNamespace = $Namespace
```

Comment:
Why change this? The call could be passing another namespace not equal to $Namespace as an argument right?